pkgs/misc/themes/gtk2/gtk-engine-murrine/default.nix

```diff
@@ -1,16 +1,23 @@
-{ stdenv, fetchurl, pkgconfig, intltool, gtk2 }:
+{ stdenv, fetchurl, pkgconfig, intltool, gtk2, gnome3 }:
 
-stdenv.mkDerivation {
-  name = "gtk-engine-murrine-0.98.2";
+stdenv.mkDerivation rec {
+  pname = "gtk-engine-murrine";
+  version = "0.98.2";
 
   src = fetchurl {
-    url = "mirror://gnome/sources/murrine/0.98/murrine-0.98.2.tar.xz";
+    url = "mirror://gnome/sources/${pname}/${stdenv.lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
     sha256 = "129cs5bqw23i76h3nmc29c9mqkm9460iwc8vkl7hs4xr07h8mip9";
   };
 
   nativeBuildInputs = [ pkgconfig ];
   buildInputs = [ intltool gtk2 ];
 
+  passthru = {
+    updateScript = gnome3.updateScript {
```

Comment:
Theme engines are GTK2 only feature and has been archived: https://gitlab.gnome.org/Archive/murrine

Reply:
I read that, too. But I saw that Ubuntu has them for GTK3 (or it's just bad naming): https://packages.ubuntu.com/search?keywords=gtk3-engines

I'll remove the `updateScript`

Reply:
Hmm, apparently the support was not removed https://blogs.gnome.org/mclasen/2014/06/13/a-new-default-theme-for-gtk/, they are just frowned upon now.

Reply:
Interesting!